lib/handlerTemplate.js

```diff
@@ -7,8 +7,8 @@
 
 var P = require('bluebird');
 var URI = require('swagger-router').URI;
+var Template = require('swagger-router').Template;
```

Comment:
Probably better to require `swagger-router` once and then set the short-cut variables.